doc/sphinxext/mne_doc_utils.py

```diff
@@ -159,6 +161,11 @@ def reset_warnings(gallery_conf, fname):
         "A worker stopped while some jobs were given to the executor.*",
         category=UserWarning,
     )
+    # neo
+    warnings.filterwarnings(
+        "ignore",
+        "The 'copy' argument in Quantity is deprecated.*",
+    )
```

Comment:
need equivalent in conftest too:

https://github.com/mne-tools/mne-python/actions/runs/10617877023/job/29431850995?pr=12820#step:17:5278